docs/resources/cluster.md

```diff
@@ -2,7 +2,7 @@
 subcategory: "Deprecated"    
 ---
 
-**WARNING:** This resource is deprecated, use `mongodbatlas_advanced_cluster`. To learn more about how to migrate, please read the [Migration Guide](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/guides/cluster-to-advanced-cluster-migration-guide).
+**WARNING:** This resource is deprecated, use `mongodbatlas_advanced_cluster`. To learn more about how to migrate, please read the [Migration Guide](../guides/cluster-to-advanced-cluster-migration-guide).
```

Comment:
Previous comments on this apply. See the top of the PR for these comments please.